Lesson_11/Tusk_1.py

```diff
@@ -0,0 +1,20 @@
+from time import ctime
+
+
+def func_decorator(func):
+    def inside_func():
+        print(f'You call the function {func}')
+        func()
+        print(f'This function was called at {ctime()}')
+    return inside_func()
```

Comment:
Do not call this wrapper since it will be executed immediately after the decorator applies. Use `return inside_func` instead